Add README to otel github org (#5)
open-telemetry/.github: README.md

<h1 align="center">
    High-quality, ubiquitous, and portable telemetry to
    enable effective observability
</h1>

<p align="center">
  <a href="https://opentelemetry.io/">
    <img src="https://opentelemetry.io/img/logos/opentelemetry-horizontal-color.svg" alt="OpenTelemetry logo" height="180">
  </a>
</p>

<p align="center">

> OpenTelemetry is a collection of tools, APIs, and SDKs. Use it to

Reviewer: ```suggestion
OpenTelemetry is a collection of APIs, SDKs, and tools. Use it to
```

Reviewer: There is an order of importance.

Author: good point, I copied that sentence from opentelemetry.io, so looks like we should change it there as well?

> instrument, generate, collect, and export telemetry data (<a href="https://opentelemetry.io/docs/concepts/signals/metrics/">metrics</a>, <a href="https://opentelemetry.io/docs/concepts/signals/logs/">logs</a> and <a href="https://opentelemetry.io/docs/concepts/signals/traces/">traces</a>) to help you analyze your software’s performance

Reviewer 2: ```suggestion
instrument, generate, collect, and export telemetry data (<a href="https://opentelemetry.io/docs/concepts/signals/metrics/">metrics</a>, <a href="https://opentelemetry.io/docs/concepts/signals/logs/">logs</a> and <a href="https://opentelemetry.io/docs/concepts/signals/traces/">traces</a>) to help you analyze your software's performance
```

and behavior.
</p>

<p align="center">
  <a href="https://opentelemetry.io/docs/what-is-opentelemetry/">Learn More 📚</a>
  ·
  <a href="https://opentelemetry.io/docs/getting-started/">Get Started 🔭</a>
  ·

>   <a href="https://github.com/open-telemetry/community/blob/main/CONTRIBUTING.md">Contribute 🫶</a>

Reviewer 3: Something is wrong with the emoji here: `Contribute 🫶`

Author: @pellared it works for me (on MacOS), can you share a screenshot of what you are seeing? I can take another one

Reviewer 3: ![image](https://github.com/open-telemetry/.github/assets/5067549/0ad82d98-bf7b-4c1f-8db0-c21b31fd513d)

(Windows 10)

Author: oh... thanks, I changed it to a different one, hope that one works :)

Reviewer 3: works 👍

  ·
  <a href="https://opentelemetry.io/docs/demo/">Try the demo 🎓</a>
</p>

---

## 👋 Getting involved

> There is a lot to do! If you are interested in getting involved, please join the [mailing lists](https://github.com/open-telemetry/community#mailing-lists) and [attend the community meetings](https://calendar.google.com/calendar/u/0/embed?src=google.com_b79e3e90j7bbsa2n2p5an5lf60@group.calendar.google.com). If you're interested in contributing to a specific part of the project, please join the appropriate [special interest group (SIG)](https://github.com/open-telemetry/community#special-interest-groups). We are a friendly, collaborative group and look forward to working together!

Reviewer 4: ```suggestion
There is a lot to do! If you are interested in getting involved, please join the [mailing lists](https://github.com/open-telemetry/community#mailing-lists) and [attend the community meetings](https://github.com/open-telemetry/community#calendar). If you're interested in contributing to a specific part of the project, please join the appropriate [special interest group (SIG)](https://github.com/open-telemetry/community#special-interest-groups). We are a friendly, collaborative group and look forward to working together!
```

I think this place would be too easy to miss in case the link would ever need to be updated (e.g. with https://github.com/open-telemetry/community/issues/1414 and https://github.com/open-telemetry/community/issues/1440), so the extra indirection might be better.

Author: good point, let me updated that!
